### DISCUSSION TITLE
How do I use this in existing project ?

### DISCUSSION THREAD

**Discussion by lnxfsf** [vikejs/vike#1870] [Discussion]
They said "zero config SSR for Vite".
But when I [read docs](https://vite-plugin-ssr.com/add)  , how to add it to existing project, I have no idea what I need to do next after `npm install vite-plugin-ssr`. 

Like, I don't know, what is `Define _default.page.client.js and _default.page.server.js` or what's `Create your first .page.js file`
And examples make absolutely no sense.

**Comment by phonzammi** [comment]
> Vite-plugin-ssr has been renamed Vike, see [migration guide](https://vite-plugin-ssr.com/vike).

Please read this guide: [Add SSR/SSG to existing Vite app](https://vike.dev/add)

**Comment by lnxfsf** [comment]
no I still don't understand what to do. 
At one point, when trying to set up, suddenly I need to set up "V1 Design", or I need X or Y.. so this is not "config free" setup at all ! 

I thought it should be simple as putting in plugins, and it automatically does stuff. 

I don't know, it seems like, I have to put all routes in /pages app ? as it differently handles routes ? 

then, I should move on Astro or NextJS, and not to fiddle with this, that even lack proper tutorial how to migrate properly. 

Is there some youtube tutorial, which explains step by step, how to migrate from React (which uses Vite), to this , so I just want this SSR feature, for better SEO..

**Reply by brillout** [reply]
https://github.com/brillout/vite-plugin-ssr_legacy/commit/a4bb4986dc38b0b5bc229899fb5b5e8f91f00cc1
